Sources/YMatterType/Elements/TypographyButton.swift

```diff
@@ -200,6 +200,7 @@ open class TypographyButton: UIButton {
         // This should be a no-op, but it's needed to force the attributed title
         // to redraw itself when button state changes
         setTitleColor(currentTitleColor, for: state)
+        adjustFonts()
```

Comment:
Can you check to see if just `restyleText()` is sufficient here? We shouldn't need to recalculate the layout and set the font.

Reply:
You're right. I just checked and it works perfectly fine.